sdk/src/main/scala/com/horizen/block/SidechainCreationVersions.scala

```diff
@@ -5,12 +5,15 @@ object SidechainCreationVersions extends Enumeration {
 
   val SidechainCreationVersion0: SidechainCreationVersion = Value(0)
   val SidechainCreationVersion1: SidechainCreationVersion = Value(1)
+  val SidechainCreationVersion2: SidechainCreationVersion = Value(2)
+  val SidechainCreationVersion3Plus: SidechainCreationVersion = Value(3)
 
   def getVersion(version: Int): SidechainCreationVersion = {
     version match {
       case 0 => SidechainCreationVersion0
       case 1 => SidechainCreationVersion1
-      case unknownVersion => throw new IllegalArgumentException(s"Unknown sidechain creation version $unknownVersion.")
```

Comment:
I think we still need to have case unknownVersion in case of argument with negative value